symbolic-debuginfo/src/dwarf.rs

```diff
@@ -394,6 +396,7 @@ impl<'d, 'a> UnitRef<'d, 'a> {
         &self,
         entry: &Die<'d, '_>,
         language: Language,
+        bcsymbolmap: Option<&BcSymbolMap<'d>>,
```

Comment:
In rust I'm API-wise usually tempted to create a `resolve_function_name_with_bcsymbolmap()` function instead of having a `None` argument to a function.